resources/public/api/conf/2.0/schemas/retrieveBalanceAndTransactions/itemDetails.json

```diff
@@ -0,0 +1,119 @@
+            "type": "boolean",
+            "example": "false"
+          },
+          "isEstimatedChargeOnHold": {
+            "description": "A clearing lock is used only for statistical or forecasted charges.",
+            "type": "boolean",
+            "example": "false"
+          },
+          "isInterestAccrualOnHold": {
+            "description": "Indicates if the charge is not accruing interest due to a review or investigation.",
+            "type": "boolean",
+            "example": "true"
+          },
+          "isInterestChargeOnHold": {
+            "description": "A lock has been put in so the interest will not be charged to the customer.",
+            "type": "boolean",
+            "example": "true"
+          },
+          "isReturn": {
+            "description": "If true, means that the charge has been generated by a Self-assessment return. ",
+            "type": "boolean",
+            "example": "true"
+          },
+          "paymentReference": {
+            "description": "A unique reference number is associated with the payment.",
+            "type": "string",
+            "example": "Ab12453535"
+          },
+          "paymentAmount": {
+            "description": "If the item relates to a payment, the payment amount is the amount of the payment received into the account.\n\nIf the item relates to a charge, the payment amount is the amount that this entry changed the transaction by. The value must be between -99999999999.99 and 99999999999.99 to 2 decimal places.",
+            "type": "number",
+            "multipleOf": 0.01,
+            "minimum": -99999999999.99,
+            "maximum": 99999999999.99,
+            "example": "5009.99"
+          },
+          "paymentMethod": {
+            "description": "The method used to make the payment.",
+            "type": "string",
+            "example": "BACS RECEIPTS"
+          },
+          "paymentLot": {
+            "description": "If the item is a payment, the payment lot is a unique identifier for a credit. If the item is a charge, the payment lot is a unique identifier for an allocation within the charge and allows you to identify where the payment came from.",
+            "type": "string",
+            "example": "81203010024"
+          },
+          "paymentLotItem": {
+            "description": "A unique ID is used in combination with a payment ID to refer to an individual payment or credit.",
+            "type": "string",
+            "example": "000001"
+          },
+          "clearingSAPDocument": {
+            "description": "When a payment is used against a charge, this links payment and charge together.",
+            "type": "string",
+            "example": "3350000253"
+          },
+          "isChargeEstimate": {
+            "description": "Estimated charges due for the customer",
+            "type": "boolean",
+            "example": "true"
+          },
+          "required": [
```

Comment:
This is in the wrong place, it should be placed after the next bracket - in line 115